src/common/view.js

```diff
@@ -8,24 +8,27 @@ export function renderView(view) {
   }
 
   view.render();
+  view._isRendered = true;
 
   if (!view.supportsRenderLifecycle) {
-    view._isRendered = true;
     view.triggerMethod('render', view);
   }
 }
 
-export function destroyView(view) {
+export function destroyView(view, disableDetachEvents) {
```

Comment:
This looks new to me, do you see this being publicly documented or is it only used internally?

Reply:
This entire module is not publicly accessible